app/design/adminhtml/default/default/layout/admin.xml

```diff
@@ -21,6 +21,14 @@
         </reference>
     </adminhtml_input_counter_handle>
 
+    <!-- admin account edit page -->
+    <adminhtml_system_account_index>
+        <reference name="head">
+            <action method="addJs"><name>maho-passkey-tools.js</name></action>
```

Comment:
@justinbeaty I created this file because those functions are not 100% base64 and they can be used only for passkey related stuff (shared between frontend and backend)

Reply:
Got it, will leave it alone. 😄

Reply:
I mean, let's say we add passkey support for frontend one day... I don't know :-D

Reply:
Yeah, that would be excellent!